graphql/image_resolver.go

```diff
@@ -0,0 +1,34 @@
+package graphql
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/evergreen-ci/evergreen/model/distro"
+	"github.com/evergreen-ci/evergreen/rest/model"
+	restModel "github.com/evergreen-ci/evergreen/rest/model"
+	"github.com/evergreen-ci/evergreen/thirdparty"
+)
+
+// Distros is the resolver for the distros field.
+func (r *imageResolver) Distros(ctx context.Context, obj *thirdparty.Image) ([]*model.APIDistro, error) {
+	if obj.ImageID == "" {
+		return nil, ResourceNotFound.Send(ctx, "unable to find imageID from provided Image")
```

Comment:
I'm not sure if not found is the correct error because this just should be passed in right?